<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartCalc_v2.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(68, 235, 153);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="res_show">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>240</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 20px;
	background-color: rgb(255, 255, 255);
	border: 2px solid gray;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color:rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_backspace">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color:rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color:rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color:rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color:rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_minus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color:rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color:rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>320</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: rgb(0, 0, 0);
	color: rgb(243, 243, 243);
	border: 1px solid gray;
	font-size: 20px;
}

QPushButton:pressed {
	background-color:rgb(46, 52, 54);
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>121</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>380</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font-size: 20px;
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
	border: 1px solid gray;
}
QPushButton:pressed {
	background-color: rgb(211, 215, 207);
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>-10</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/images/sber.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../misc/images/sber.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_expand">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
background-image: url(:/resource/images/calc.png);
color: rgb(238, 238, 236);
border: 2px solid gray;
font-size: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
background-image: url(:/resource/images/graph.png);
color: rgb(238, 238, 236);
border: 2px solid gray;
font-size: 15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_credit_calc">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/images/credit.png);
border: 2px solid gray;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_deposit_calc">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/resource/images/deposit.png);
border: 2px solid gray;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 11pt &quot;Uroob&quot;;
color: rgb(255, 255, 255);
font-size: 32px;</string>
    </property>
    <property name="text">
     <string>SmartCalc_v2.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_x">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>66</y>
      <width>41</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;
background-color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_gran">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>51</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;
background-color: rgb(243, 243, 243);</string>
    </property>
    <property name="text">
     <string>gran</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
